code/modules/vehicles/motorized_wheelchair.dm

```diff
@@ -0,0 +1,168 @@
+		return
+	return ..()
+	
+/obj/vehicle/ridden/wheelchair/motorized/attackby(obj/item/I, mob/user, params)
+	if(I.tool_behaviour == TOOL_SCREWDRIVER)
+		I.play_tool_sound(src)
+		panel_open = !panel_open
+		user.visible_message("<span class='notice'>[user] [panel_open ? "opens" : "closes"] the maintenance panel on [src].</span>", "<span class='notice'>You [panel_open ? "open" : "close"] the maintenance panel.</span>")
+		return
+	if(panel_open)
+		if(istype(I, /obj/item/stock_parts))
+			if(istype(I, /obj/item/stock_parts/cell))
+				if(power_cell)
+					to_chat(user, "<span class='warning'>There is a power cell already installed.</span>")
+				else
+					I.forceMove(src)
+					power_cell = I
+					to_chat(user, "<span class='notice'>You install the [I].</span>")
+				refresh_parts()
+				return
+			var/obj/item/stock_parts/B = I
+			var/P 
+			for(var/obj/item/stock_parts/A in contents)
+				for(var/D in required_parts)
+					if(ispath(A.type, D))
+						P = D
+						break
+				if(istype(B, P) && istype(A, P))
+					if(B.get_part_rating() > A.get_part_rating())
+						B.forceMove(src)
+						user.put_in_hands(A)
+						user.visible_message("<span class='notice'>[user] replaces [A] with [B] in [src].</span>", "<span class='notice'>You replace [A] with [B].</span>")
+						break
+			refresh_parts()
+			return
+	return ..()
+
+/obj/vehicle/ridden/wheelchair/motorized/wrench_act(mob/living/user, obj/item/I)
+	to_chat(user, "<span class='notice'>You begin to detach the wheels...</span>")
+	if(I.use_tool(src, user, 40, volume=50))
+		to_chat(user, "<span class='notice'>You detach the wheels and deconstruct the chair.</span>")
+		new /obj/item/stack/rods(drop_location(), 8)
+		new /obj/item/stack/sheet/metal(drop_location(), 10)
+		var/turf/T = get_turf(src)
+		for(var/atom/movable/A in contents)
+			A.forceMove(T)
+			if(isliving(A))
+				var/mob/living/L = A
+				L.update_mobility()
+		qdel(src)
+	return TRUE
+
+/obj/vehicle/ridden/wheelchair/motorized/examine(mob/user)
+	. = ..()
+	if((obj_flags & EMAGGED) && panel_open)
+		. += "There is a bomb under the maintenance panel."
+	. += "There is a small screen on it, [(in_range(user, src) || isobserver(user)) ? "[power_cell ? "it reads:" : "but it is dark."]" : "but you can't see it from here."]"
+	if(!power_cell || (!in_range(user, src) && !isobserver(user)))
+		return
+	. += "Speed: [speed]"
+	. += "Energy efficiency: [power_efficiency]"
+	. += "Power: [power_cell.charge] out of [power_cell.maxcharge]"
+
+/obj/vehicle/ridden/wheelchair/motorized/Bump(atom/movable/M)
+	. = ..()
+	// Here is the shitty emag functionality.
+	if(obj_flags & EMAGGED && (istype(M, /turf/closed) || isliving(M)))
+		explosion(src, -1, 1, 3, 2, 0)
+		visible_message("<span class='boldwarning'>[src] explodes!!</span>")
+		return
+	// If the speed is higher than 6.7 throw the person on the wheelchair away
+	if(M.density && speed > 6.7 && has_buckled_mobs())
```

Comment:
is there a runspeed define you can use here?

Reply:
I see you use it again below, definitely should be a define of some sort if not already

Reply:
the value is from wheelchairs, I have no idea where it is from but it is used to divide run speed to get the vehicle move delay.

Reply:
cringe

Reply:
oh well, I will name it to something like speed divider and put it to wheelchair.

Reply:
anyways I made it a delay_multiplier var and not a define so there isn't an pretty much useless define just floating around.